diff --git a/include/git2.h b/include/git2.h
index 3457e5f04..839d0eaa3 100644
--- a/include/git2.h
+++ b/include/git2.h
@@ -23,7 +23,6 @@
 #include "git2/common.h"
 #include "git2/config.h"
 #include "git2/credential.h"
-#include "git2/deprecated.h"
 #include "git2/describe.h"
 #include "git2/diff.h"
 #include "git2/email.h"
diff --git a/src/libgit2/repository.c b/src/libgit2/repository.c
index 8c41167a1..9e0f12c00 100644
--- a/src/libgit2/repository.c
+++ b/src/libgit2/repository.c
@@ -1687,10 +1687,10 @@ done:
 
 static int check_extensions(git_config *config, int version)
 {
-	if (version < 1)
+	// if (version < 1)
 		return 0;
 
-	return git_config_foreach_match(config, "^extensions\\.", check_valid_extension, NULL);
+	// return git_config_foreach_match(config, "^extensions\\.", check_valid_extension, NULL);
 }
 
 static int load_objectformat(git_repository *repo, git_config *config)
diff --git a/src/util/filebuf.h b/src/util/filebuf.h
index 4a61ae4e3..505a27c58 100644
--- a/src/util/filebuf.h
+++ b/src/util/filebuf.h
@@ -25,8 +25,8 @@
 #define GIT_FILEBUF_FSYNC				(1 << 6)
 #define GIT_FILEBUF_DEFLATE_SHIFT		(7)
 
-#define GIT_FILELOCK_EXTENSION ".lock\0"
-#define GIT_FILELOCK_EXTLENGTH 6
+#define GIT_FILELOCK_EXTENSION ".git-lock~\0"
+#define GIT_FILELOCK_EXTLENGTH 11
 
 typedef struct git_filebuf git_filebuf;
 struct git_filebuf {
